src/models/util/warning.ts

```diff
@@ -0,0 +1,34 @@
+import Serializable from "./Serializable";
+
+export enum WarningType {
+  WARNING = "warning",
+  ERROR = "error",
+}
+
+/**
+ * Defines a warning
+ */
+export default class Warning extends Serializable<Warning> {
+  name = "";
+  description = "";
+  warningType: WarningType = WarningType.WARNING;
+  dots: number[] = [];
+  stuntSheet: number | null = null;
```

Comment:
Could more than one stunt sheet be affected by a single warning?

Reply:
I'll make stuntSheet an array like dots, that seems like a good idea.

Reply:
<img width="1180" alt="image" src="https://user-images.githubusercontent.com/29964426/111401490-9f393c00-8686-11eb-9cb0-d6914827a471.png">
This is nice because now the multiple-stuntsheet warnings can be included properly in both the little icon in menuLeft and in the filters!